### DISCUSSION TITLE
Difference between diameq:lognormal(1,0.05) and 1-sphericity:lognormal(0.145,0.03)?

### DISCUSSION THREAD

**Discussion by Geo572** [neperfepx/neper#747] [Q&A] [answered]
Hello @mkasemer @rquey,
What is the difference between diameq:lognormal(1,0.05) and 1-sphericity:lognormal(0.145,0.03), because both show lognormal(mean, standard deviations)? 
Its better to explain the difference between these two commands.

Thank you.

**Comment by rquey** [comment, marked as answer]
`diameq:lognormal(1,0.05)` applies to the cell sizes (equivalent diameter) while `1-sphericity:lognormal(0.145,0.03)` applies to the cell shapes (1 - sphericity).

**Reply by Geo572** [reply]
Thank you Professor Dr. Romain Quey for your response .
